components/kyma-metrics-collector/pkg/process/process.go

```diff
@@ -383,6 +417,15 @@ func (p *Process) populateCacheAndQueue(runtimes *kebruntime.RuntimesPage) {
 				Metric:          nil,
 			}
 
+			// record kebFetchedClusters metric for trackable cluster
+			recordKEBFetchedClusters(
+				true,
```

Comment:
Please create a new const and use it here:

```golang
const trackable = true
```

Reply:
Done